.github/workflows/thurloe-build-tag-publish.yml

```diff
@@ -88,6 +88,7 @@ jobs:
       id-token: 'write'
 
   set-version-in-dev:
+    if: ${{ github.ref_name == 'develop' }}
```

Comment:
Do you want to only do this on merge to `develop` or on merge to the repo's default branch. While it's unlikely that the default branch will change, if you were to say, copy-paste this GHA to another repo it could create the same kind of issue we're trying to mitigate here.